config/sota-local.toml

```diff
@@ -10,6 +10,3 @@ path = "storage"
 
 [pacman]
 type = "none"
-
-[uptane]
-secondary_config_file = "virtualsec.json"
```

Comment:
Why remove this? I think it's just used for testing and as an example.

Reply:
Since recently the initialization fails if this file doesn't exist. It seems like a correct behavior, but makes this config unusable without a secondary config. I'd really like to keep this config as a minimal example. Maybe we can create another one which will include a secondary.